src/frontend/data/context/reducer.ts

```diff
@@ -17,16 +17,16 @@ export interface ContextState<state> {
   };
 }
 
-export const initialState = {
+export const initialState = (appData: AppData) => ({
```

Comment:
I think this should be named `initState` or `buildInitialState` now that it's a function instead of an object